core/web/api/investigation.py

```diff
@@ -99,14 +100,18 @@ def bulk_add(self, id):
 
         try:
             for node in data['nodes']:
-                if node['type'] in globals() and issubclass(
-                        globals()[node['type']], Observable):
-                    _type = globals()[node['type']]
-
-                n = _type.get_or_create(value=node['value'])
-                if node['new_tags']:
-                    n.tag(node['new_tags'].split(', '))
-                nodes.append(n)
+                try:
+                    if node['type'] in globals() and issubclass(
+                            globals()[node['type']], Observable):
+                        _type = globals()[node['type']]
+
+                    n = _type.get_or_create(value=node['value'])
+                    if node['new_tags']:
+                        n.tag(node['new_tags'].split(', '))
+                    nodes.append(n)
+                except Exception as e:
```

Comment:
It would be good to catch a more specific type of exception here. Are you looking for `ObservableValidationError`?

Reply:
thanks for point for better solution, updated